take/src/polsl/take/restaurant/api/OrderREST.java

```diff
@@ -56,5 +58,11 @@ public void deleteOrder(@PathParam("id") int id) {
 	public List<Meal> getOrderMeals(@PathParam("order_id") int order_id){
 		return orderService.findOrderMeals(order_id);
 	}
+	
+	@POST
+	@Path("/newOrder")
+	public Order addOrderByRequest(OrderRequest orderRequest){
+		return orderService.addOrderByRequest(orderRequest);
+	}

```

Comment:
![image](https://user-images.githubusercontent.com/43828389/129446999-31b913c5-dcbb-42ba-9bf3-0774f9c62dfd.png)

Leci 405 method not allowed